<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalVariablesWidget</class>
 <widget class="QWidget" name="GlobalVariablesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="tree_view"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottom_panel">
     <item>
      <widget class="QPushButton" name="refresh_button">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filter_box">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="group_by_section">
       <property name="text">
        <string>Group by section</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="group_by_source_file">
       <property name="text">
        <string>Group by source file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
